LICENSE.md

```diff
@@ -26,13 +30,16 @@ Version 2.0.3
 
 **Modpack Creator**: An individual who creates modpacks.
 
-**Author(s)**: Original programmer or programmers responsible for writing the Mod.
+**Author(s)**: Original programmer(s), texturer(s), and contributors responsible for creating, in part or in whole, the Mod.
```

Comment:
This license is mostly intended as a code and compiled binary license, not an art license. This section could use clarification, but more specific is probably not the direction to go. The less specific the language used (eg contributor rather than "person who submits a pull request via github") the better the license is able (to a certain extent) to protect the person using it.